sentry/src/test/java/io/sentry/config/FilesystemPropertiesLoaderTest.kt

```diff
@@ -0,0 +1,51 @@
+package io.sentry.config
+
+import io.sentry.NoOpLogger
+import java.nio.charset.Charset
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+import org.junit.rules.TemporaryFolder
+
+class FilesystemPropertiesLoaderTest {
+
+    var folder = TemporaryFolder()
```

Comment:
This should be JUnit 4 `@Rule` but either it does not work with `kotlin-test` or something else is broken with our test setup that makes Gradle unable to run tests with rules.

Reply:
in Kotlin you need to use: `@get:Rule var ...`

eg

```
@get:Rule
var mActivityRule: ActivityTestRule<MainActivity> = ActivityTestRule(MainActivity::class.java)
```

Reply:
Thanks!